Mustachio/ParserOptions.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.IO;
+using System.Text;
+
+namespace Mustachio
+{
+	/// <summary>
+	/// Options for Parsing run
+	/// </summary>
+	public class ParserOptions
```

Comment:
There should be an object instead of single parameters to allow easy extension of more features without more breaking changes